fighters/common/src/function_hooks/get_param.rs

```diff
@@ -214,6 +214,15 @@ pub unsafe fn get_param_float_hook(x0 /*boma*/: u64, x1 /*param_type*/: u64, x2
             }
         }
 
+        else if fighter_kind == *FIGHTER_KIND_MEWTWO {
```

Comment:
Would prefer this be done within status script

Reply:
is there another example of someone doing that and is it cleaner than plugging it in there?

Reply:
turning the flag off didnt do anything